src/main/java/com/mongodb/kafka/connect/source/MongoSourceConfig.java

```diff
@@ -184,15 +184,70 @@ public class MongoSourceConfig extends AbstractConfig {
   public static final String TOPIC_NAMESPACE_MAP_CONFIG = "topic.namespace.map";
   private static final String TOPIC_NAMESPACE_MAP_DISPLAY = "The namespace to topic map";
   private static final String TOPIC_NAMESPACE_MAP_DOC =
-      "A json map that maps change stream document namespaces to topics.\n"
-          + "For example: `{\"db\": \"dbTopic\", \"db.coll\": \"dbCollTopic\"}` will map all "
-          + "change stream documents from the `db` database to `dbTopic.<collectionName>` apart from"
-          + "any documents from the `db.coll` namespace which map to the `dbCollTopic` topic.\n"
-          + "If you want to map all messages to a single topic use `*`: "
-          + "For example: `{\"*\": \"everyThingTopic\", \"db.coll\": \"exceptionToTheRuleTopic\"}` "
-          + "will map all change stream documents to the `everyThingTopic` apart from the `db.coll` "
-          + "messages."
-          + "Note: Any prefix and suffix configuration will still apply.";
+      "A JSON object configuring how to map a MongoDB change stream document namespace"
```

Comment:
The current online documentation for `topic.namespace.map` is here: https://www.mongodb.com/docs/kafka-connector/current/source-connector/configuration-properties/kafka-topic/.